packages/mobile/src/escrow/saga.ts

```diff
@@ -139,6 +140,13 @@ export function* transferToEscrow(action: EscrowTransferPaymentAction) {
     // Tranfser the funds to the Escrow contract.
     Logger.debug(TAG + '@transferToEscrow', 'Transfering to escrow')
     yield call(registerStandbyTransactionLegacy, context, amount.toString(), escrowWrapper.address)
+    yield call(
+      registerStandbyTransaction,
+      context,
+      amount.toString(),
```

Comment:
I haven't checked but I think we might need to negate this as well?

Reply:
Yep, it makes sense :)